plugins/janus_audiobridge.c

```diff
@@ -6663,6 +6663,9 @@ static void *janus_audiobridge_mixer_thread(void *data) {
 				}
 				ps = ps->next;
 			}
+
+			g_list_free(anncs_list);
+			anncs_list = NULL;
```

Comment:
No need to set the pointer to NULL, since it's a locally defined variable. Please also remove the extra unneeded empty line above.